tools/cloud-build/daily-tests/integration-group-4.yaml

```diff
@@ -107,3 +107,26 @@ steps:
     ansible-playbook tools/cloud-build/daily-tests/ansible_playbooks/slurm-integration-test.yml \
       --user=sa_106486320838376751393 --extra-vars="project=${PROJECT_ID} build=$${BUILD_ID_SHORT}" \
       --extra-vars="@tools/cloud-build/daily-tests/tests/slurm-v5-ubuntu.yml"
+
```

Comment:
We usually also update at line 15, a list of the tests in this group for easy visibility. If that is now off sync, we may want to remove it entirely or update it.

Reply:
Updated.